perception/image_projection_based_fusion/docs/roi-pointcloud-fusion.md

```diff
@@ -0,0 +1,93 @@
+# roi_pointcloud_fusion
+
+## Purpose
+
+The node `roi_pointcloud_fusion` is to cluster the pointcloud based on Region Of Interests (ROIs) detected by a 2D object detector, specific for unknown labeled ROI.
+
+## Inner-workings / Algorithms
+
+- The pointclouds are projected onto image planes and extracted as cluster if they are inside the unknown labeled ROIs.
+- Since the cluster might contain unrelated points from background, then a refinement step is added to filter the background pointcloud by distance to camera.
+
+![roi_pointcloud_fusion_image](./images/roi_pointcloud_fusion.png)
+
+## Inputs / Outputs
+
+### Input
+
+| Name                     | Type                                                     | Description                                               |
+| ------------------------ | -------------------------------------------------------- | --------------------------------------------------------- |
+| `input`                  | `sensor_msgs::msg::PointCloud2`                          | input pointcloud                                          |
+| `input/camera_info[0-7]` | `sensor_msgs::msg::CameraInfo`                           | camera information to project 3d points onto image planes |
+| `input/rois[0-7]`        | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | ROIs from each image                                      |
+| `input/image_raw[0-7]`   | `sensor_msgs::msg::Image`                                | images for visualization                                  |
+
+### Output
+
+| Name                     | Type                                                     | Description                               |
+| ------------------------ | -------------------------------------------------------- | ----------------------------------------- |
+| `output`                 | `sensor_msgs::msg::PointCloud2`                          | output pointcloud as default of interface |
+| `output_clusters`        | `tier4_perception_msgs::msg::DetectedObjectsWithFeature` | output clusters                           |
+| `~/debug/image_raw[0-7]` | `sensor_msgs::msg::Image`                                | images for visualization                  |
```

Comment:
[Q] Does this node output a debug image? It seems like the code is outputting a point cloud.

Reply:
@miursh 
[A] Thank you, You're right. Since I keep debug mode of fusion interface as default, so debug image will not be published. 
In additional, `debug/clusters` topic is added to visualize the clustered pointcloud.
The docs is update here: https://github.com/autowarefoundation/autoware.universe/pull/4681/commits/75aeedb4c121d5b5e34d336775c9f9d7de96f4d3